.github/workflows/clang-format-fix.yml

```diff
@@ -1,15 +1,15 @@
-name: clang-format Check
+name: clang-format Commit Changes
```

Comment:
This is the most important line. Both clang-format-fix and clang-format-check should not use the same name.